app/js/lib/view_handler/ppl_recurrent_warning.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	/**
+	 * View Handler for showing and hiding elements if the update value matches a regular expression
+	 * @class
+	 */
+	PPLRecurrentWarning = {
+		element: null,
+		isHidden: null,
+
+		update: function ( formContent ) {
+
+			if ( formContent.paymentType === 'PPL' && formContent.paymentIntervalInMonths > 0 ) {
+				if ( this.isHidden === false ) {
+					return;
+				}
+				this.element.show();
+				this.isHidden = false;
+			} else {
+				if ( this.isHidden === true ) {
+					return;
+				}
+				this.element.hide();
+				this.isHidden = true;
+			}
```

Comment:
Is there an issue with calling `show` or `hide` on an element that already has the respective state? Like this

```js
this.isHidden = formContent.paymentType !== 'PPL' || formContent.paymentIntervalInMonths === 0
if ( this.isHidden ) {
    this.element.hide();
}
else {
    this.element.show();
}
```

Of course its even simpler if `this.element` has some `setVisibility`.